code/modules/clothing/modular_armor/attachments/modules.dm

```diff
@@ -250,38 +250,12 @@
 	variants_by_parent_type = list(/obj/item/clothing/suit/modular/xenonauten = "mod_ff_xn")
 
 /obj/item/armor_module/module/chemsystem
-	name = "Vali chemical enhancement module"
-	desc = "Designed for mounting on modular armor. This experimental module runs on green blood taken from xenos with harvester class weapons; Green blood heals the user and boosts any chems in the suit injection system. \nUse the suit menu to connect harvester class weapons, control the injection system, find chem boost information, and more."
+	name = "Vali Health Scan System"
```

Comment:
We can probably remove the "Vali", since Vali is on it's way out.
Keep it simple, "Health Scan System" and it's fine like that.